apps/picsa-tools/seasonal-calendar-tool/src/app/app-routing.module.ts

```diff
@@ -7,6 +7,16 @@ export const ROUTES_COMMON: Routes = [
     loadChildren: () => import('./pages/home/home.module').then((m) => m.HomeModule),
     title: 'Seasonal Calendar',
   },
+  {
+    path: 'create-calendar',
```

Comment:
nit(non-blocking)
The url will already have /seasonal-calendar/ so probably can just call the child routes /create and /[calendarId] to view

Reply:
Alright, thanks for this @chrismclarke